deploy-board/deploy_board/webapp/cluster_view.py

```diff
@@ -355,6 +360,19 @@ def get_base_images(request):
     })
 
 
+def get_base_image_events(request, image_id):
+    base_images_events = baseimages_helper.get_image_events_by_newId_with_result(
+        request, image_id)
+
+    goldens = baseimages_helper.get_image_tag_by_id(request, image_id)
```

Comment:
`tags` would be more generic to `goldens`.

Reply:
ok, sounds good